openadapt/utils.py

```diff
@@ -156,11 +174,13 @@ def get_monitor_dims():
 
 
 def draw_ellipse(
-    x, y, image,
-    width_pct=.03,
-    height_pct=.03,
-    fill_transparency=.25,
-    outline_transparency=.5,
+    x,
+    y,
+    image,
+    width_pct=0.03,
+    height_pct=0.03,
+    fill_transparency=0.25,
+    outline_transparency=0.5,
```

Comment:
These should go to `.config`, maybe put a TODO for now (for here and elsewhere in this file, but one TODO is sufficient)